cpp/src/ray/test/benchmark/benchmark.cc

```diff
@@ -0,0 +1,124 @@
+#include <gtest/gtest.h>
+#include <ray/api.h>
+
+#include <chrono>
+#include <set>
+
+using namespace ::ray::api;
+
+void NoReturn() {}
+int ReturnVal() { return 1; }
+
+void Input(std::string str) {}
+
+std::string Echo(std::string str) { return str; }
+
+RAY_REMOTE(NoReturn, ReturnVal, Input, Echo);
+
+class Counter {
+ public:
+  int count;
+
+  Counter(int init) { count = init; }
+
+  static Counter *FactoryCreate() { return new Counter(0); }
+
+  int Add() {
+    count += 1;
+    return count;
+  }
+
+  int Get() const { return count; }
+};
+
+RAY_REMOTE(Counter::FactoryCreate, &Counter::Add, &Counter::Get);
+
+template <typename F>
+void Execute(F f, std::string func_name, size_t num_repeats) {
```

Comment:
Can you give an clear name of this function? I can't get the difference between `Execute` and `ExecuteLatancy`.